README.rst

```diff
@@ -240,52 +240,6 @@ Getting Started
 * `Frequently Asked Questions`_
 * Contributing_
 
-What is eBPF and XDP?
-=====================
-
-Berkeley Packet Filter (BPF) is a Linux kernel bytecode interpreter originally
-introduced to filter network packets, e.g. for tcpdump and socket filters. The
-BPF instruction set and surrounding architecture have recently been
-significantly reworked with additional data structures such as hash tables and
-arrays for keeping state as well as additional actions to support packet
-mangling, forwarding, encapsulation, etc. Furthermore, a compiler back end for
-LLVM allows for programs to be written in C and compiled into BPF instructions.
-An in-kernel verifier ensures that BPF programs are safe to run and a JIT
-compiler converts the BPF bytecode to CPU architecture-specific instructions
-for native execution efficiency. BPF programs can be run at various hooking
-points in the kernel such as for incoming packets, outgoing packets, system
-calls, kprobes, uprobes, tracepoints, etc.
-
-BPF continues to evolve and gain additional capabilities with each new Linux
-release. Cilium leverages BPF to perform core data path filtering, mangling,
-monitoring and redirection, and requires BPF capabilities that are in any Linux
-kernel version 4.8.0 or newer (the latest current stable Linux kernel is
-4.14.x).
```

Comment:
v4.14 was an LTS, supported for six years and retired last year. Mentioning it as the latest is clearly a little out-of-date :D